### DISCUSSION TITLE
Thumbfast not working since MPV 0.37.x

### DISCUSSION THREAD

**Discussion by yuka123** [mpv-player/mpv#14510] [General]
https://github.com/po5/thumbfast

It's generating black thumbnails only, ever since MPV 0.37.x update. It's working fine on the 0.36.x. 

I thought it might be [this](
https://github.com/mpv-player/mpv/commit/86b498ecc0e6276bb905df57f479d0506b45ba53), but I can't find any reference to that removed property in the .lua.

Does anybody know what could be the issue? 

Thanks!

**Comment by geextahslex** [comment]
Is there a specific reason you prefer thumbfast over [thumbnail_script](https://github.com/marzzzello/mpv_thumbnail_script)?

**Comment by yuka123** [comment]
I am unsure how to make thumbnail_script work with uosc? Should it work out of the box by just placing lua? Because I tried and it didn't work. I'll re-check

**Reply by geextahslex** [reply]
I don't know, I don't use uosc. Are there no updates on https://github.com/tomasklaen/uosc?

**Reply by yuka123** [reply]
I don't think so and the thumbnail_script seems to only work on the default osd bars.. :(

**Comment by yuka123** [comment]
Ok I found out the issue is caused by ThumbFast option for Tone Mapping:

vo_gpu: remove --tone-mapping-mode

Seems the removed tone mapping mode from 0.37 changelog. How does one use tone mapping now since 0.37? Imean in the script.

**Reply by geextahslex** [reply]
you are not using gpu-next?

**Reply by yuka123** [reply]
I am, I got vo=gpu-next in the config. Still doesn't work with tone-mapping on in thumbfast config.
